postcss.config.js

```diff
@@ -26,7 +30,7 @@ const tw = {
 };
 
 const config = {
-  // add postcss plugins, required to extend apollo tailwind config
+  // Add postcss plugins, required to extend apollo tailwind config
```

Comment:
This doesn't actually make much sense to me here because you are not adding postcss plugin, you are adding tailwindcss plugin. :)

Reply:
it says postcss plugins not plugin tho 🤔

Reply:
Comment is referring to: ```plugins: [require('tailwindcss')(tw)],```

Reply:
Doh! Just noticed this plugins is part of postcss.config.js haha. Please disregard!